plutus-playground-server/usecases/CrowdFunding.hs

```diff
@@ -255,3 +258,11 @@ This part of the API (the PendingTx argument) is experimental and subject
 to change.
 
 -}
+
+myCurrency :: KnownCurrency
+myCurrency = KnownCurrency "b0b0" "MyCurrency" ( "USDToken" :| ["EURToken"])
+$(mkKnownCurrencies ['myCurrency])
+
+iotsDefinitions :: Text
```

Comment:
will this replace the `mkFunctions` function?

Reply:
No, it's going to get rolled in, asap. :-}